src/components/Form/Form.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
+import { View, StyleSheet, Text, TextInput, Image, TouchableOpacity } from 'react-native';
+
+class Form extends Component {
+  state = {
+      category: '',
+      description: '',
+      loaction: '',
+      photo: ''
+    }
+
```

Comment:
Unless I am not seeing something we are supposed to use hooks for this project if we are using react native. We might need to change these over to useState. I am working on confirming that now but i'm pretty sure.